README.md

```diff
@@ -97,7 +102,34 @@ The SDO owner services are packaged as a single docker container that can be run
    ```bash
    curl -sS -w "%{http_code}" -X POST $SDO_RV_URL/mp/113/msg/20 | jq
    ```
+   
+### <a name="gen-keypair"></a>Generate Owner Key Pairs
+
+If you want to expedite the process of creating key pairs, you can run `keys/generate-key-pair.sh`
+To run this script you must be using Ubuntu. 
+
+1. Go to the directory where you want your generated keys to be saved then download `generate-key-pair.sh`, which is used to create key pairs for Owner Attestation:
+
+   ```bash
+   curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/master/keys/generate-key-pair.sh
+   chmod +x generate-key-pair.sh
+   ```
+   
+2. Run `generate-key-pair.sh` script. If this process is being ran for a non production environment you **do not** need to create your own keys. 
+You will be prompted to answer a few questions in order to produce corresponding certificates to your private keys:
+
+   ```bash
+   ./generate-key-pair.sh
+   ```
 
+3. After running `generate-key-pair.sh` you will have created and been left with two files.
+- `owner-keys.tar.gz`: A tar file containing the 3 private keys and associated 3 certifications
+- `Owner-Public-Key.pem`: Device customer/owner public key. This is needed to extend the voucher to the owner. Either pass it as an argument in `simulate-mfg.sh` or give this public key to a device manufacturer.
```

Comment:
I know i said "or" when we were talking yesterday, but now that i read it i think it should be "and", because they will likely do both. I suggest: Pass this file as an argument whenever running `simulate-mfg.sh` and give this public key to each device manufacturer producing SDO-enabled devices for you.